RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerWeights.cc

```diff
@@ -9,8 +9,10 @@
 #include "CondFormats/DataRecord/interface/EcalWeightXtalGroupsRcd.h"
 #include "CondFormats/DataRecord/interface/EcalTBWeightsRcd.h"
 
-#include <FWCore/ParameterSet/interface/ConfigurationDescriptions.h>
```

Comment:
edits in this file are outdated (different from what went to the release; before the PR review).
Merging locally suggests that there are no merge conflicts and a correct version is picked up.
Better clean up though